### PR TITLE
[14.0][ADD] stock_exception

### PR DESCRIPTION
**Pull request by newtratip**

### CODE REVIEW

File: stock_exception/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            self.ignore_exception = False
+
+    def action_confirm(self):
+        if self.detect_exceptions() and not self.ignore_exception:
+            return self._popup_exceptions()
+        return super().action_confirm()
+
+    @api.model
+    def _get_popup_action(self):
+        action = self.env.ref("stock_exception.action_stock_exception_confirm")
```
Comment (rousseldenis) [inline stock_exception/models/stock.py:48]:
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id("stock_exception.action_stock_exception_confirm")
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Looks great. One change

**Comment by rousseldenis** [comment]
@newtratip Could you attend comments ?

File: stock_exception/models/stock.py (hunk 1 above)
Comment (newtratip) [reply stock_exception/models/stock.py:48]: Fixed. may i ask, why change `self.env["ir.actions.act_window"]._for_xml_id(...)` is better `self.env.ref(...)` ?

Comment (newtratip) [reply stock_exception/models/stock.py:48]: @rousseldenis, I used this code `self.env["ir.actions.act_window"]._for_xml_id(...)` is error, I rollback the code.
Selection_735

Comment (rousseldenis) [reply stock_exception/models/stock.py:48]: Ok, I didn't know the base_exception behaviour.

It does the sudo().read. But that's that part which is not correct (in base exception).

Of course you can keep your code.

**Review by Saran440 — APPROVED** [review]
LGTM, Functional Test on Production :+1:

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**